libtsuba/test/storage-format-version/v5-optional-datastructure-rdklsh-index.cpp

```diff
@@ -0,0 +1,129 @@
+#include <cstdint>
```

Comment:
Once this lands and katana-enterprise is updated lets make sure to get at least these tests in:
1. LSH index created, saved, and loaded (just like this test)
2. RDG with existing LSH loaded (this will need to be updated with every change to the index implementation)
3. Test emulating case where an RDG contains an outdated LSH index and a load index is attempted from it ( to mimic graceful failing). BTW, right now in the present implementation this case will cause an std::abort.